communityApp/API/Interest.js

```diff
@@ -0,0 +1,18 @@
+import { INTEREST_TYPE_FETCH_URL } from "../constants/Database";
+
+const path = 'interest_type/';
+
+/**
+ * Get the list of interest types.
+ * It does not require access token for authentication since it will be used for signup the process.
+ * 
+ * @returns Json object containing the list of interest types
+ */
+export async function getInterestTypes() {
+    const message = {
+    // Empty message will be used for postRequest() function since it requires some object
```

Comment:
??? This is empty

Reply:
It will send the empty message because postRequest() function in BaseResquest has to contain some message object even if there is no data to pass to backend.